aeppl/transforms.py

```diff
@@ -413,6 +415,19 @@ def log_jac_det(self, value, *inputs):
         return -at.log(value)
 
 
+class InverseTransform(RVTransform):
+    name = "inverse"
+
+    def forward(self, value, *inputs):
+        return at.inv(value)
+
+    def backward(self, value, *inputs):
+        return at.inv(value)
+
+    def log_jac_det(self, value, *inputs):
+        return -at.log(value**2)
```

Comment:
```suggestion
        return -2 *at.log(value)
```

I wonder if this is something we could have as a specialization rewrite or is one of those that is not universally better